genre.html

```diff
@@ -41,34 +41,34 @@ <h1>Select a Genre</h1>
     <div class="row">
       <div class="selection-container">
         <div class="select-button-div">
-          <button data-value="workout" class="select-button">Workout</button>
+          <button data-type="genre" data-value="workout" class="select-button">Workout</button>
         </div>
         <div class="select-button-div">
-          <button data-value="toplists" class="select-button">Top Hits</button>
+          <button data-type="genre" data-value="toplists" class="select-button">Top Hits</button>
         </div>
         <div class="select-button-div">
-          <button data-value="pop" class="select-button">Pop</button>
+          <button data-type="genre" data-value="pop" class="select-button">Pop</button>
         </div>
         <div class="select-button-div">
-          <button data-value="hiphop" class="select-button">Hip-Hop</button>
+          <button data-type="genre" data-value="hiphop" class="select-button">Hip-Hop</button>
         </div>
         <div class="select-button-div">
-          <button data-value="rnb" class="select-button">RnB</button>
+          <button data-type="genre" data-value="rnb" class="select-button">RnB</button>
         </div>
         <div class="select-button-div">
-          <button data-value="rock" class="select-button">Rock</button>
+          <button data-type="genre" data-value="rock" class="select-button">Rock</button>
         </div>
         <div class="select-button-div">
-          <button data-value="indie_alt" class="select-button">Indie</button>
+          <button data-type="genre" data-value="indie_alt" class="select-button">Indie</button>
         </div>
         <div class="select-button-div">
-          <button data-value="chill" class="select-button">Chill</button>
+          <button data-type="genre" data-value="chill" class="select-button">Chill</button>
         </div>
         <div class="select-button-div">
-          <button data-value="anime" class="select-button">Anime</button>
+          <button data-type="genre" data-value="anime" class="select-button">Anime</button>
         </div>
         <div class="select-button-div">
-          <button data-value="edm_dance" class="select-button">EDM</button>
+          <button data-type="genre" data-value="edm_dance" class="select-button">EDM</button>
         </div>
       </div>
     </div>
```

Comment:
I forgot to call this out on the previous feature, but please use JavaScript view swapping instead of multiple HTML files: https://github.com/Learning-Fuze/ajax-project-instructions#indexhtml

---

![image](https://user-images.githubusercontent.com/16143771/157160611-8300566e-3413-47a1-9e64-d215df54a21f.png)